### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by fabiozaramella**

### CODE REVIEW

**Comment by fabiozaramella** [comment]
Ahaha next time i shouldn't do it as soon as i wake up 😅️

File: po/extra/it.po [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-03 06:39+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Fabio Zaramella\n"
+"Language-Team: none\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
```
Comment (ryonakano) [inline po/extra/it.po]:
```suggestion
"Content-Type: text/plain; charset=UTF-8\n"
```

The CI complains this :sweat_smile:

**Review by ryonakano — CHANGES_REQUESTED** [review]

File: po/extra/it.po (hunk 1 above)
Comment (fabiozaramella) [reply po/extra/it.po]: yup, sorry

**Comment by fabiozaramella** [comment]
WTF, it saved the same file twice, i'll update it once again. Buggy Poedit 🤦🏻️

**Comment by fabiozaramella** [comment]
Nevermind, I guess I have to update it once again after the last merge. I'll do it later. 👍🏽️

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+"Language-Team: \n"
+"X-Generator: Poedit 2.3\n"
+
+#: src/Indicator.vala:76
+msgid "IBus Daemon is Not Running"
+msgstr "Il demone IBus non è in esecuzione"
+
+#: src/Indicator.vala:77
+msgid "Click \"Input Method Settings…\" to show available input method engines."
+msgstr "Fai clic su \"Impostazioni metodi di inserimento\" per visualizzare i metodi disponibili."
```
Comment (ryonakano) [inline po/it.po]:
```suggestion
msgstr "Fai clic su \"Impostazioni metodi di inserimento…\" per visualizzare i metodi disponibili."
```

I don't understand Italian, but don't you need the trailing ellipsis as the original sentence and the translation in the line 30 have?

Comment (fabiozaramella) [reply po/it.po]: Of course, thanks for noticing! 😅

**Review by ryonakano — APPROVED** [review]
Glad to see this project is now available in English, Japanese, plus Italian! :tada:

**Pull request merged**